sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveQl.scala

```diff
@@ -520,6 +520,7 @@ https://cwiki.apache.org/confluence/display/Hive/Enhanced+Aggregation%2C+Cube%2C
             "TOK_TBLTEXTFILE", // Stored as TextFile
             "TOK_TBLRCFILE", // Stored as RCFile
             "TOK_TBLORCFILE", // Stored as ORC File
+            "TOK_TBLPARQUETFILE", // Stored as PARQUET
```

Comment:
@yhuai  Thanks for reply, The token can  be parsed in Hive 12  too,  but the ddl "create table ... stored as parquet " will be failed when call hive api.
This is why i do not know how to add a test case that only running in hive-0.13.1